creme/linear_model/__init__.py

```diff
@@ -5,6 +5,7 @@
 from .pa import PAClassifier
 from .pa import PARegressor
 from .softmax import SoftmaxRegression
+from .adpredictor import AdPredictor
```

Comment:
Can you rename the module to `ad_predictor`?

Reply:
Yes !